static/app/views/performance/http/queryParameterDecoders/panel.tsx

```diff
@@ -0,0 +1,15 @@
+import {decodeScalar} from 'sentry/utils/queryString';
+
+const DEFAULT_PANEL = 'duration';
+type Panel = 'duration' | 'status';
+
+export default function decode(value: string | string[] | undefined | null): Panel {
+  switch (decodeScalar(value)) {
+    case 'duration':
+      return 'duration';
+    case 'status':
+      return 'status';
+    default:
+      return DEFAULT_PANEL;
+  }
+}
```

Comment:
Copied this pattern from existing decoders. A little verbose, but it's tucked away